src/topp/IDFilter.cpp

```diff
@@ -149,16 +146,15 @@ class TOPPIDFilter :
     registerOutputFile_("out", "<file>", "", "output file ");
     setValidFormats_("out", ListUtils::create<String>("idXML"));
 
-    registerTOPPSubsection_("precursor", "Filtering by precursor RT or m/z");
+    registerTOPPSubsection_("precursor", "Filtering by precursor attributes (RT, m/z, charge, length)");
     registerStringOption_("precursor:rt", "[min]:[max]", ":", "Retention time range to extract.", false);
     registerStringOption_("precursor:mz", "[min]:[max]", ":", "Mass-to-charge range to extract.", false);
+    registerStringOption_("length", "[min]:[max]", ":", "Keep only peptide hits with a sequence length in this range.", false);
+    registerStringOption_("charge", "[min]:[max]", ":", "Keep only peptide hits with charge states in this range.", false);
```

Comment:
this should probably change to `precursor:charge` and `precursor:length` since it will display correctly (and make sense?)